packages/web-components/src/text/text.options.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * TextSize constants
+ * @public
+ */
+export const TextSize = {
+  _100: '100',
+  _200: '200',
+  _300: '300',
+  _400: '400',
+  _500: '500',
+  _600: '600',
+  _700: '700',
+  _800: '800',
+  _900: '900',
+  _1000: '1000',
+} as const;
+
+/**
+ * The type for TextSize
+ * The font size and line height based on the theme tokens
+ * @public
+ */
+export type TextSize = typeof TextSize[keyof typeof TextSize];
+
+/**
+ * TextFont Constants
+ * @public
+ */
+export const TextFont = {
+  base: 'base',
+  numeric: 'numeric',
+  monospace: 'monospace',
+} as const;
+
+/**
+ * Applies font family to the content
+ * @public
+ */
+export type TextFont = typeof TextFont[keyof typeof TextFont];
+
+/**
+ * TextWeight Constants
+ * @public
+ */
+export const TextWeight = {
+  medium: 'medium',
+  regular: 'regular',
+  semibold: 'semibold',
+} as const;
```

Comment:
React implementation supports bold as well:
https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-text/src/components/Text/Text.types.ts#L71-L76

Reply:
Ah this was a miss!